ibis/expr/types/geospatial.py

```diff
@@ -671,6 +671,25 @@ def transform(self, srid: ir.IntegerValue) -> GeoSpatialValue:
         """
         return ops.GeoTransform(self, srid).to_expr()
 
+    def convert(
+        self, source_crs: ir.StringValue, target_crs: ir.StringValue | ir.IntegerValue
```

Comment:
Can call these `source` and `target`? Since people seem to interchange CRS and SRID, I think it might be less confusing if there's no specific nomenclature attached to the argument names.

This would also make it easier to map to backends, which may or may not use the CRS or SRID terminology.

Reply:
That makes sense. I'll get it in a few min